fixed extratorrent search not working v.3.3.7 #5736

yezezey opened the pull request:
Fixed https://github.com/qbittorrent/qBittorrent/issues/5736
Tested for both python2 and python3.

thalieht commented:
Tested :+1:

yezezey commented:
@thalieht
Thanks for testing.
Should I increase the version number of this plugin?

thalieht commented:
I don't know

yezezey commented:
OK, better leave sledgehammer999 to decide it.
thanks.

sledgehammer999 commented:
@yezezey please use interactive rebase and `fixup` the 2nd commit into the 1st one. Also rename the first one to capitalize the first letter of the first word. Also put `Closes #5736` at the end. When I merge it will autoclose that bug.

Also tagging past plugin contributors @DoumanAsh and @ngosang

yezezey commented:
@sledgehammer999 
Did I make it?

sledgehammer999 commented:
Nitpicking: You don't need the dash. It should read like sentence. However, you don't actually need to re-rebase it if you're bored/lazy.

PS: I'll wait a couply of days if the tagged people want to comment something. Then I'll merge.

yezezey commented:
yup, i am lazy :)

DoumanAsh commented:
///Tried github reviews... But i'll leave my opinion as comment in thread
Looks ok, but i would appreciate leaving details how it's got fixed in commit message...

yezezey commented:
How about this? Please review before I push.

```
Fixed extratorrent search not working Closes #5736 #5753
The original code accumulates all the <td> texts between the description link and the number of seeders into self.current_item["size"]:
    <td>248.72</td><td>&nbsp;MB<td> => "248.72 MB".
ExtraTorrents changed the table design, added the Added column and join the number and its unit:
    <td>8d</td><td>248.72&nbsp;MB</td> => "8d 248.72 MB"
This commit store only the last <td> text in self.current_item["size"]. 
```

@DoumanAsh @ngosang
I had never used ExtraTorrents before the design change, so the above description is based on my guess. Do you remember how it had been?

DoumanAsh commented:
@yezezey Actually i think there was also issue of utf-8 producing multiple pieces, but right now i cannot be sure.
Overall intention and reasoning of your change is clear.

Regardless small suggestion:
Leave empty line after "Fixed extratorrent search not working Closes #5736 #5753" and detailed description.

yezezey commented:
@DoumanAsh
 Thanks for the suggestion.

@sledgehammer999
 I deleted v.3.3.7 from the first line because plugins are not related any specific version of qbt.

sledgehammer999 commented:
Thanks for this.
